inference-engine/tests/functional/plugin/cpu/single_layer_tests/topk.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "functional_test_utils/ov_tensor_utils.hpp"
```

Comment:
minor
can we add test cases when on each inference we got different `k` value
could you add test cases with defined upper bound?

it's not blocked for merge

Reply:
Test cases with upper bounds have been already added.
Test cases with different `k` on each inference was added.
Thank you!